tests/end_to_end/features/provider_updates.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Provider Updates
+
+  Scenario Outline: Rotate S3 Credentials
+    Given Featureform is installed
+    When I create a "hosted" "insecure" client for "localhost:7878"
+    And I create the AWS credentials with an access key "<access_key>" and secret key "<secret_key>"
+    And I register S3 with name "<name>", bucket_name "<bucket_name>", bucket_region "<bucket_region>", path "<path>"
+    And I create the AWS credentials with an access key "<repl_access_key>" and secret key "<repl_secret_key>"
```

Comment:
Should this be done before the register S3?

Reply:
These tests are testing updates, so it recreates the credentials again